plugins/services/src/js/containers/pod-debug/PodDebugContainer.tsx

```diff
@@ -223,7 +223,7 @@ class PodDebugTabView extends React.Component {
       primaryAction = (
         <Trans render="span">
           <a
-            href={MetadataStore.buildDocsURI("/services#service-status")}
+            href={MetadataStore.buildDocsURI("/gui/services#service-status")}
```

Comment:
Can you explain why we need this change?

Reply:
because the docs-team seems to have rearranged the structure without setting up redirects.